tests/util.go

```diff
@@ -123,12 +146,18 @@ func WaitUntilResourceReadyByLabelTestNamespace(resourceType, label, query, expe
 
 func CreateUser(username string) {
```

Comment:
This function should return an `error`. Otherwise, there is no way how to analyze whether it was executed successfully.

Reply:
I have updated with the changes